CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ BUG FIXES:
 * resource/cloudflare_argo: call `Read` for `Import` operations ([#1295](https://github.com/cloudflare/terraform-provider-cloudflare/issues/1295))
 * resource/cloudflare_argo_tunnel: update CNAME to use `cfargotunnel.com` ([#1293](https://github.com/cloudflare/terraform-provider-cloudflare/issues/1293))
 * resource/cloudflare_origin_ca_certificate: reintroduce `DiffSuppressFunc` for `requested_validity` changes to handle all schema/SDK combinations ([#1289](https://github.com/cloudflare/terraform-provider-cloudflare/issues/1289))
+* resource/cloudflare_split_tunnel: fix importing ([#1313](https://github.com/cloudflare/terraform-provider-cloudflare/issues/1313))
```

Comment:
the CHANGLOG is automatically generated; please see [changelog-process](https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/docs/changelog-process.md) for details on populating this entry.

Reply:
yep. i realized my mistake just after posting the PR... my bad.  should be fixed now.